packages/lesswrong/server/emailComponents/EmailInactiveUserSummary.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { Fragment } from "react";
+import { defineStyles, useStyles } from "@/components/hooks/useStyles";
+import { postGetPageUrl } from "@/lib/collections/posts/helpers";
+import { getEAEmojiByName } from "@/lib/voting/eaEmojiPalette";
+import { getSiteUrl } from "@/lib/vulcan-lib/utils";
+import { EmailUsername } from "./EmailUsername";
+
+export type BestReaction = {name: string, count: number};
+
+const styles = defineStyles("EmailInactiveUserSummary", (theme: ThemeType) => ({
+  root: {
+    background: theme.palette.text.alwaysWhite,
+    color: theme.palette.text.alwaysBlack,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 14,
+    fontWeight: 400,
+    lineHeight: "22px",
+    width: 522,
+    maxWidth: "100%",
+    margin: "24px auto",
+  },
+  spacing: {
+    marginBottom: 48,
+  },
+  heading: {
+    fontSize: 20,
+    fontWeight: 700,
+  },
+  mostCommentedPost: {
+    textDecoration: "underline",
+    color: "inherit",
+  },
+  notifications: {
+    fontSize: 12,
+    textDecoration: "underline",
+  },
+  post: {
+    marginBottom: 20,
+    padding: "12px 24px",
+    background: theme.palette.grey[100],
+    borderRadius: theme.borderRadius.default,
+  },
+  postTitle: {
+    fontSize: 32,
+    fontWeight: 700,
+    lineHeight: "36px",
+    textDecoration: "none",
+    marginBottom: 0,
+    color: "inherit",
+  },
+  postAuthors: {
+    marginBottom: 16,
+  },
+  postAuthor: {
+    fontWeight: 500,
+    color: theme.palette.text.alwaysBlack,
+    textDecoration: "none",
+  },
+  postBody: {
+    fontWeight: 500,
+  },
+  hr: {
+    border: "none",
+    borderBottom: `1px solid ${theme.palette.text.alwaysBlack}`,
+  },
+}));
+
+const truncateDescription = (description: string = "") => {
+  const truncated = description.slice(0, 260);
+  return truncated.length < description.length ? truncated + "..." : truncated;
+}
+
+export const EmailInactiveUserSummary = ({
+  user,
+  karmaChange,
+  bestReaction,
+  mostCommentedPost,
+  unreadNotifications,
+  recommendedPosts,
+}: {
+  user: DbUser,
+  karmaChange: number,
+  bestReaction?: BestReaction,
+  mostCommentedPost?: {post: DbPost, commentCount: number},
+  unreadNotifications: number,
+  recommendedPosts: PostsList[],
+}) => {
+  const bestEmoji = bestReaction ? getEAEmojiByName(bestReaction.name) : null;
+  const showKarmaChange = karmaChange > 0;
+  const showBestReaction = bestReaction && bestReaction.count > 0 && bestEmoji;
+  const showMostCommentedPost = mostCommentedPost && mostCommentedPost.commentCount > 0;
+  const showUnreadNotifications = unreadNotifications > 0 && unreadNotifications < 5;
+
+  const showNotificationsSection =
+    showKarmaChange ||
+    showBestReaction ||
+    showMostCommentedPost ||
+    showUnreadNotifications;
+
+  const classes = useStyles(styles);
+  return (
+    <div className={classes.root}>
+      <p>
+        Hi <EmailUsername user={user} />,
```

Comment:
Actually could you remove the link here and just display the username? I noticed it felt a bit odd to be linked to my own profile in the email